tests/unit/dataactvalidator/test_scripts_loadFile.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+from dataactcore.models.domainModels import CGAC
+from dataactvalidator.scripts import loadFile
+from tests.unit.dataactcore.factories.domain import CGACFactory
+
+
+def test_delete_missing_cgacs(database):
```

Comment:
+1 for testing our load files, moving in a great direction, and fast, with our testing